components/inputs/test/input-time-range.test.js

```diff
@@ -103,7 +103,8 @@ describe('d2l-input-time-range', () => {
 			expect(elem.endValue).to.equal('00:30:00');
 		});
 
-		it('should update startValue as expected when set through property', async() => {
+		// timing out in legacy-Edge via GitHub Actions
+		it.skip('should update startValue as expected when set through property', async() => {
```

Comment:
These tests are timing out in legacy-Edge on BrowserStack but only from GitHub Actions... super weird. :(

Reply:
We had the `karma-browserstack-launcher` version locked at 1.4.0 before whereas it looks like it's not anymore - maybe that's somehow related?

Reply:
Thanks, I'll give that a try. Although even if it resolves the issue I'm not sure if it's worth holding back on an old version just for legacy-Edge. We'll see.

Reply:
Agreed

Reply:
Still failing on 1.4.0. I think they [fixed the issues](https://github.com/karma-runner/karma-browserstack-launcher/pull/158/files) with post-1.4.0 timeouts, so this must be a different thing. What's weird is the error message:

> Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

But according to our config, our mocha timeout should be 10 seconds. 🤷